### DISCUSSION TITLE
Email when comments are added

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1600] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X ] Reviewed the existing feature requests and my feature has not been requested.
- [X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
There's currently no way to know when a comment is added on a recipe, unless you stumble upon it.

**(Optional): Suggest A Solution**
Mealie should send an email to all users when a comment is added to a recipe. It'd be ideal if there was an option to control which actions cause an email to be received.

I'm not sure if this was already considered with https://github.com/hay-kot/mealie/issues/833 but it's not yet listed there, unless that counts as updating the recipe?

**Additional Information**
_FILL THESE OUT_
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
We could probably add an event listener for comments, although I think we'd probably want its own database entry otherwise you'll get all recipe updates which isn't what you're looking for.

Thoughts on that, @hay-kot? Apprise supports emails so that part should be simple.

**Comment by fivestones** [comment]
Maybe I should start a new feature request for this, but it would also be great to have a feed inside of mealie itself that listed things like comments that have been added to recipies, probably with multiple ways to sort them but at least by default  listing in order of whatever comments were most recently added. Then if people are talking about a recipe, you can easily see that when you open mealie and you can easily join it or make that recipe. I think this would make sharing recipes with friends much more collaborative and fun, and you'd have the option to see new comments only in mealie--something I would like more than to have my email clogged with lots of meail comment emails (at least as an option).

**Reply by michael-genson** [reply]
Mealie already has this more or less on the global timeline, you can see everything that's been created, made, etc. And you can include pictures now so you can share how it turned out

Currently it doesn't do anything with comments, though that would be pretty easy to implement on the timeline if you want to submit a feature request

**Reply by michael-genson** [reply]
https://github.com/mealie-recipes/mealie/pull/1831
https://github.com/mealie-recipes/mealie/pull/2265
https://github.com/mealie-recipes/mealie/pull/2444
